### ISSUE TITLE
hangs at shutdown

### ISSUE THREAD

**Issue by asarubbo** [ConsoleKit2/ConsoleKit2#95]
I'm using gentoo, and I run kde-plasma-5.8.6

Sometimes, when I poweroff the machine form the kde menu, it hangs and I need to shutdown the pc from the power button.
The one line that I see in the logs refers to consolekit.
```
/var/log/kern.log:Jun 29 19:25:05 localhost kernel: [  587.168103] traps: ck-remove-direc[16719] trap int3 ip:7f6b604f2d61 sp:7ffe2a78faf0 error:0
```

Do you have hints on how to properly debug the error?

Consolekit-1.1.0

**Comment by EricKoegel** [comment]
It was fixed in Consolekit-1.1.1, best bet would be to update to that or 1.1.2.

**Comment by asarubbo** [comment]
After several days of testing I think that 1.1.2 definitely solves the issue
